resources/views/components/hidden-controls.blade.php

```diff
@@ -0,0 +1,28 @@
+<script>
+    function toggleHiddenControls() {
+        const buttonEl = document.getElementById('hidden-controls-toggle-button');
+        const contentEl = document.getElementById('hidden-controls-content');
+        if (contentEl) {
+            if (contentEl.classList.contains('hidden')) {
+                contentEl.classList.remove('hidden');
+                if (buttonEl) {
+                    buttonEl.innerHTML = buttonEl.innerText.substring(0, buttonEl.innerText.length-1) + "▲";
+                }
+            } else {
+                contentEl.classList.add('hidden');
+                if (buttonEl) {
+                    buttonEl.innerHTML = buttonEl.innerText.substring(0, buttonEl.innerText.length-1) + "▼";
+                }
+            }
+        }
+    }
```

Comment:
```suggestion
    function toggleHiddenControls() {
        const buttonEl = document.getElementById('hidden-controls-toggle-button');
        const contentEl = document.getElementById('hidden-controls-content');
        
        if (contentEl && buttonEl) {
            contentEl.classList.toggle('hidden');
            buttonEl.innerText = buttonEl.innerText.slice(0, -1) + (contentEl.classList.contains('hidden') ? "▼" : "▲");
        }
    }
```

A bit more concise. I believe this is functionally equivalent, though the `buttonEl` conditional is lifted up higher.

Reply:
This was based on the function you added for the Moderate button on the profile page:
https://github.com/wescopeland/RAWeb/blob/fd5d5c789a90c7e179ff5f8f08ce5bd45ae11430/resources/views/components/user/profile/primary-meta.blade.php#L114-L124

I'll make the suggested changes.